### PR TITLE
New Rule: Reply-to/Sender Mismatch with suspicious TLD

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

File: detection-rules/headers_replyto_mismatch_sus_tld.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+name: "Reply-to/Sender Mismatch with suspicious TLD"
+description: |
+  This rule detects a mismatch between the reply-to and the sender email addresses, and one or both of them are from suspicious TLDs
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    any(headers.reply_to,
+        .email.email != sender.email.email
```
Comment (jkamdjou) [inline detection-rules/headers_replyto_mismatch_sus_tld.yml:10]: do we wanna check if the domains don't match too? that'd prevent us from firing on stuff like this:

From: sam@foo.agency
Reply-to: support@foo.agency

**Comment by morriscode** [comment]
This can be reworked now that we have suspicious tld list in a better place

**Comment by morriscode** [comment]
This is firing an unwanted amount of times. Closing

**Pull request closed without merging**